.github/workflows/issue-metrics.yml

```diff
@@ -2,13 +2,52 @@ name: Measure issue timestamp
 
 on:
   issues:
+<<<<<<< HEAD
```

Comment:
Once you make the other changes I have mentioned, hit this PR with one more rebase and I think we will be rid of the issue metrics stuff 🙂